sturdycouscous/Garbanzo/requirements.txt

```diff
@@ -0,0 +1,42 @@
+zope.interface==5.2.0
+wheel==0.35.1
+websockets==8.1
+w3lib==1.22.0
+urllib3==1.26.2
+Twisted==20.3.0
+soupsieve==2.0.1
+six==1.15.0
+setuptools==50.3.2
+service-identity==18.1.0
+Scrapy==2.4.1
+requests==2.25.0
+queuelib==1.5.0
+pyOpenSSL==20.0.0
+pymongo==3.11.1
+PyHamcrest==2.0.2
+PyDispatcher==2.0.5
+pycparser==2.20
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+Protego==0.1.16
+pip==20.2.4
+parsel==1.6.0
+lxml==4.6.2
+jmespath==0.10.0
+itemloaders==1.0.4
+itemadapter==0.2.0
+incremental==17.5.0
+idna==2.10
+hyperlink==20.0.1
+hiredis==1.1.0
+cssselect==1.1.0
+cryptography==3.2.1
+constantly==15.1.0
+chardet==3.0.4
+cffi==1.14.4
+certifi==2020.11.8
+beautifulsoup4==4.9.3
+Automat==20.2.0
+attrs==20.3.0
+async-timeout==3.0.1
+aioredis==1.3.1
```

Comment:
I there I thought I was the one adding too many external dependencies ><

Reply:
should requirements.txt be a directory up?

Reply:
It was a short list before I did a `pip freeze` to pin the versions of the original 7 packages: https://github.com/IsabelleLenertz/sturdy-couscous/pull/20/commits/79908e587239d0138e5ec71791272c4ba96fc16d